src/RA_Dlg_AchievementsReporter.cpp

```diff
@@ -45,34 +45,39 @@ void Dlg_AchievementsReporter::SetupColumns(HWND hList)
     ms_nNumOccupiedRows = 0;
 }
 
-_Use_decl_annotations_
-void Dlg_AchievementsReporter::AddAchievementToListBox(HWND hList, const Achievement* restrict pAch)
+_Use_decl_annotations_ void Dlg_AchievementsReporter::AddAchievementToListBox(HWND hList,
+                                                                              const Achievement* restrict pAch)
 {
     Expects(pAch != nullptr);
+    int check = 0;
     // We aren't actually using the value so we're using iterators
     for (auto it = COL_TITLE.cbegin(); it != COL_TITLE.cend(); ++it)
     {
         const auto nPos{std::distance(COL_TITLE.cbegin(), it)};
+
         switch (ra::itoe<Column>(nPos))
         {
             case Column::Checked:
-                sprintf_s(LbxDataAt(ms_nNumOccupiedRows, nPos), MAX_TEXT_LEN, "%s", "");
+                check = sprintf_s(LbxDataAt(ms_nNumOccupiedRows, nPos), MAX_TEXT_LEN, "%s", "");
```

Comment:
Since they array uses `char*` instead of `std::string`, we just checked the value.